### ISSUE TITLE
Example data field mismatch

### ISSUE THREAD

**Issue by andrawaag** [weso/SAF-Lux#13]
Currently, the example data is provided as a two-dimensional data frame, where multiple dimensions are stored with field separators (i.e. ";") within a single cell. This is error-prone because the structure is implicit. We can only assume that the field order is consistent across fields.

In the example, data already contains such an inconsistency : 
`['Willibrordus', 'Willibrordus', 'Clemens Willibrordus', 'Willibrordus', 'Willibrord', 'Willibrord', 'Willibrord']
['Ultraiectensis', 'Traiectensis', 'Sanctus', 'Saint', 'Heiliger', 'Sankt']`

For the 7 alternative names, there are only 6 titles.

**Comment by mzeinstra** [comment]
Good catch Andra,

I think it is ok to have this uploaded with the assumption that the order is consistent. With a SHeX of the CIDOC we can build propoer ingest tooling based on an commonly accepted RDF structure.

**Comment by joelthill** [comment]
after review with Martine:

![temp](https://user-images.githubusercontent.com/10212884/125482921-c3af6b38-238d-4dc7-aa94-db97d5e3735f.jpg)

there are indeed 7 occurences of alternative names of which only 6 have a title. see screenshot. It drills basically down to an export challenge, because for Clemens Willibrordus, there is no empty $c, but simply no $c subfield at all. The question is: do we fix those cases (we detected roundabout 28 cases) in the test file or in the uploaded wiki data? We understand that you will have to do at least another upload due to the missing dates, don't you?

**Comment by mzeinstra** [comment]
@andrawaag see above, do you want this fixed in the test set?

@joelthill I assume this will only be a problem for the test set, since in my mind you will be using the ShEx to structure the export information in a CIDOC-model. Right?

**Comment by joelthill** [comment]
yes, I share your view cc. the initial dataset. we'll be able to fix it in the Excel sheet if this makes sense for a future upload.

**Comment by andrawaag** [comment]
Using two-dimensional data frames to transport multidimensional records, will always lead to these types of issues. So I am a bit reluctant here. But defining a yaml or json alternative will take much more time. 
I would not work with assumptions on order here, because that can lead to similar issues. The current script compares the number of separated fields and does not do anything if there is a mismatch between those counts. An easy fix is maybe to add a None value (e.g. -1, nan or None) if there is no value. In that case, the script finds an equal number of subfields and act accordingly (i.e. don't add a none-value on its designated place)
